--- CMakeLists.txt	2016-10-03 10:38:41.642712670 +0200
+++ CMakeLists.txt	2016-10-03 10:38:59.918665586 +0200
@@ -44,10 +44,7 @@
 include_directories(${ZLIB_INCLUDE_DIR})
 
 if(NOT WIN32)
-  find_library(M_LIBRARY
-    NAMES m
-    PATHS /usr/lib /usr/local/lib
-  )
+  set(M_LIBRARY "-lm")
   if(NOT M_LIBRARY)
     message(STATUS "math lib 'libm' not found; floating point support disabled")
   endif()
