--- CMakeLists.txt	2012-07-10 02:36:56.000000000 +0200
+++ CMakeLists.txt	2012-07-17 13:52:50.477403317 +0200
@@ -30,19 +30,19 @@
 find_package(ZLIB REQUIRED)
 include_directories(${ZLIB_INCLUDE_DIR})
 
-if(NOT WIN32)
-  find_library(M_LIBRARY
-    NAMES m
-    PATHS /usr/lib /usr/local/lib
-  )
-  if(NOT M_LIBRARY)
-    message(STATUS
-      "math library 'libm' not found - floating point support disabled")
-  endif()
-else()
-  # not needed on windows
-  set(M_LIBRARY "")
-endif()
+# if(NOT WIN32)
+#   find_library(M_LIBRARY
+#     NAMES m
+#     PATHS /usr/lib /usr/local/lib
+#   )
+#   if(NOT M_LIBRARY)
+#     message(STATUS
+#       "math library 'libm' not found - floating point support disabled")
+#   endif()
+# else()
+#   # not needed on windows
+#   set(M_LIBRARY "")
+# endif()
 
 # COMMAND LINE OPTIONS
 if(DEFINED PNG_SHARED)
@@ -91,7 +91,7 @@
 endif()
 
 # SET LIBNAME
-set(PNG_LIB_NAME png${PNGLIB_MAJOR}${PNGLIB_MINOR})
+set(PNG_LIB_NAME png)
 
 # to distinguish between debug and release lib
 set(CMAKE_DEBUG_POSTFIX "d")
